mlprimitives/candidates/timeseries.py

```diff
@@ -27,8 +27,8 @@ def rolling_window_sequences(X, window_size, value_column, time_column):
     time = []
     for i in range(len(X) - window_size):
         # reshape into a vector to fit into a neural network model (vectorize it)
-        output_X.append(X[i: i + window_size][value_column].values.reshape([-1, 1]))
-        y.append(X[i + window_size + 1][value_column].values.reshape([-1, 1]))
+        output_X.append(X[i: i + window_size][value_column].values.copy().reshape([-1, 1]))
```

Comment:
Why have been these `copy` statements added here?
This doesn't seem to be part of any of the issues that this PR is trying to solve.
Please review if this can be removed so that there are no changes to this file.